src/src/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -598,16 +593,16 @@ private AuthenticationResult processTokenResponse(HttpWebResponse webResponse) {
         case HttpURLConnection.HTTP_UNAUTHORIZED:
             try {
                 result = parseJsonResponse(webResponse.getBody());
-            } catch (final JSONException jsonException) {
+            } catch (JSONException jsonException) {
```

Comment:
1) to keep consistent since nowhere we catch final exception
2) and I don't really see a big point in this
Do you see any advantage to have final here?